javascript/index.js

```diff
@@ -4,37 +4,163 @@
 // 🚨🚨🚨 Comment out the below code before you start working on the code
 
 // Out of sync
-  getInstruction("mashedPotatoes", 0, (step1) => {
+/*getInstruction("mashedPotatoes",0,(step1) => {
     document.querySelector("#mashedPotatoes").innerHTML += `<li>${step1}</li>`;
-  }, (error) => console.log(error));
-  
-  getInstruction("mashedPotatoes", 1, (step2) => {
+  },
+  (error) => console.log(error)
+);*/
+
+/*getInstruction("mashedPotatoes",1,(step2) => {
     document.querySelector("#mashedPotatoes").innerHTML += `<li>${step2}</li>`;
-  }, (error) => console.log(error));
-  
-  getInstruction("mashedPotatoes", 2, (step3) => {
+  },
+  (error) => console.log(error)
+);*/
+
+/*getInstruction("mashedPotatoes",2,(step3) => {
     document.querySelector("#mashedPotatoes").innerHTML += `<li>${step3}</li>`;
-  }, (error) => console.log(error));
-  
-  getInstruction("mashedPotatoes", 3, (step4) => {
+  },
+  (error) => console.log(error)
+);*/
+
+/*getInstruction("mashedPotatoes",3,(step4) => {
     document.querySelector("#mashedPotatoes").innerHTML += `<li>${step4}</li>`;
-  }, (error) => console.log(error));
-  
-  getInstruction("mashedPotatoes", 4, (step5) => {
+  },
+  (error) => console.log(error)
+);
+
+getInstruction("mashedPotatoes",4,(step5) => {
     document.querySelector("#mashedPotatoes").innerHTML += `<li>${step5}</li>`;
     document.querySelector("#mashedPotatoesImg").removeAttribute("hidden");
-  }, (error) => console.log(error));
-
-
+  },
+  (error) => console.log(error)
+);*/
 
 // Iteration 1 - using callbacks
 // ...
+mashedPotatoes.push ("Mashed potatoes are ready!");
```

Comment:
Instead of doing this, you could've done this on step5 before removing the hidden attribute:

``document.querySelector("#mashedPotatoes").innerHTML += `<li>Mashed potatoes are ready!</li>`;``